content/en/blog/_posts/2025-04-09-ensure-secret-pulled-images/index.md

```diff
@@ -0,0 +1,145 @@
+---
+layout: blog
+title: "Image Pull Policy the way you always thought it worked!"
+date: ???
+draft: true
+slug: ensure-secret-pulled-images-alpha
+author: >
+  [Ben Petersen](https://github.com/benjaminapetersen) (Microsoft),
+  [Stanislav Láznička](https://github.com/stlaz) (Microsoft)
+---
+
+## Image Pull Policy the way you always thought it worked!
+
+Some things in Kubernetes are surprising, and the way `imagePullPolicy` behaves might
+be one of them. Given Kubernetes is all about running pods, it may be surprising
```

Comment:
vs using surprising in two sentences in a row...

```suggestion
be one of them. Given Kubernetes is all about running pods, it may be eye-opening
```

Reply:
Agree, updating!